ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -0,0 +1,57 @@
+from turtle import st
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+#bincount
+
+
+@handle_frontend_test(
+    fn_tree="numpy.bincount",
+    dtypes_values_casting=np_frontend_helpers.dtypes_values_casting_dtype(
+        arr_func=[
+            lambda: helpers.dtype_and_values(
+                available_dtypes=helpers.get_dtypes("float"),
+            )
+        ],
+        get_dtypes_kind="float",
+    ),
+    where=np_frontend_helpers.where(),
+    number_positional_args=np_frontend_helpers.get_num_positional_args_ufunc(
+            fn_name="bincount"
+        ),
+    )
+def test_numpy_bincount(
+        dtypes_values_casting,
+        where,
+        frontend,
+        test_flags,
+        fn_tree,
+        on_device,
+    ):
+        input_dtypes, x, casting, dtype = dtypes_values_casting
+        where, input_dtypes, test_flags = np_frontend_helpers.handle_where_and_array_bools(
+            where=where,
+            input_dtype=input_dtypes,
+            test_flags=test_flags,
+        )
+        np_frontend_helpers.test_frontend_function(
+            input_dtypes=input_dtypes,
+            frontend=frontend,
+            test_flags=test_flags,
+            fn_tree=fn_tree,
+            on_device=on_device,
+            x=x[0],
+            out=None,
+            where=where,
+            casting=casting,
+            order="K",
+            dtype=dtype,
+            subok=True,
+        )
```

Comment:
You should be generating and passing the arguments of the function; `x`, `weights`, and `minlength`. I would suggest you have a look at `test_jax_numpy_bincount`.

Reply:
This is still not correct. You shouldn't be passing None to arguments. You need to generate them in the test handler. Please use this as reference https://github.com/unifyai/ivy/blob/72b5158e133287851ac897b9cd0484cd2f3ded4e/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py#L196.